packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js

```diff
@@ -111,10 +122,22 @@ GeneralInformation.propTypes = {
     }),
     loadSystemDetail: PropTypes.func,
     store: PropTypes.any,
-    writePermissions: PropTypes.bool
+    writePermissions: PropTypes.bool,
+    SystemCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    OperatingSystemCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    BiosCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    InfrastructureCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    ConfigurationCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    CollectionCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ])
```

Comment:
Why `bool`? 🤔

Reply:
So consumers can rutn off the wrapper.
```JSX
<GeneralInformation
  store={useStore()}
  OperatingSystemCardWrapper={false}
  BiosCardWrapper={false}
  InfrastructureCardWrapper={false}
  ConfigurationCardWrapper={false}
  CollectionCardWrapper={false}
/>
```

Will show just bios card

Reply:
hmmm I would rather use `null` (= `Node`)

Reply:
Yeah, that works as well.

Reply:
Yeah, I understand, I just don't like the false/bool here.. 😄  but it doesn't matter too much.